pkg/application/vault/vault.go

```diff
@@ -0,0 +1,79 @@
+package vault
+
+import (
+	"github.com/awslabs/eksdemo/pkg/application"
+	"github.com/awslabs/eksdemo/pkg/cmd"
+	"github.com/awslabs/eksdemo/pkg/installer"
+	"github.com/awslabs/eksdemo/pkg/template"
+)
+
+// https://developer.hashicorp.com/vault/tutorials/kubernetes/kubernetes-raft-deployment-guide
+// https://developer.hashicorp.com/vault/tutorials/kubernetes/kubernetes-amazon-eks
+
+// fmt.Println(getField(&v, "X"))
+
+func NewApp() *application.Application {
+	app := &application.Application{
+		Command: cmd.Command{
+			Name:        "vault",
+			Description: "HashiCorp Vault Secrets and Encryption Management System",
+		},
+
+		Installer: &installer.HelmInstaller{
+			ChartName:     "vault",
+			ReleaseName:   "vault",
+			RepositoryURL: "https://helm.releases.hashicorp.com",
+			ValuesTemplate: &template.TextTemplate{
+				Template: valuesTemplate,
+			},
+		},
+	}
+
+	app.Options, app.Flags = newOptions()
+
+	return app
+
+}
+
```

Comment:
Can you add the following comment here so contributors and maintainers can easily lookup the values file?
```
// https://github.com/hashicorp/vault-helm/blob/main/values.yaml
```

Reply:
Added; keeping conversation open in case it should be positioned elsewhere in the file.

Reply:
yeah, I thought where I clicked to leave the comment would be easy to see the exact line number but it wasn't.  Can you put this comment right above:
```
const valuesTemplate = `---
```

Reply:
Done!